x/wasm-storage/types/wasm.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	fmt "fmt"
+	"strings"
+
+	"github.com/hyperledger/burrow/crypto"
+)
+
+// MaxWasmSize is the maximum size of Wasm bytecode.
+const MaxWasmSize = 800 * 1024
```

Comment:
I'm not sure so correct me if i'm wrong but isn't this better to have as a genesis config thing? So we can always upgrade it through a gov decision

Reply:
Yea that was actually my initial idea but got lazy 😋

Reply:
Hahaha, no worries, atleast it's noted